crmsh/main.py

```diff
@@ -309,7 +309,11 @@ def compgen():
 def parse_options():
     opts, args = option_parser.parse_known_args()
     utils.check_space_option_value(opts)
+    debug_value_from_opt = opts.debug and not config.core.debug
     config.core.debug = "yes" if opts.debug else config.core.debug
+    if debug_value_from_opt:
+        config.set_option("core", "debug", "yes", internal=True)
+        atexit.register(config.back_to_default)
```

Comment:
An exit hook looks complicated. Can we just filter out those internal options before saving?

Reply:
> An exit hook looks complicated. Can we just filter out those internal options before saving?

I've tried this way, but since crmsh called `config.save()` just after init_ssh, if I filtered out like `core.debug`, then further debugging messages will missing while running `crm -d cluster init -y`

Reply:
Changed again.

When saving core.hosts, filter out core.debug, and set it back after save